auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -348,7 +373,7 @@ section:not(:last-child) {
   padding-bottom: 0;
 }
 
-.continuation-authorization .col-sm-9 {
+.file-section .col-sm-9 {
   // adjust second column to vertically line up with first column
   margin-top: -6px;
```

Comment:
Unfortunately there is no helper class that gives us this style, so my best practise above kind of falls apart here. I usually just do this anyway. You could also create your own "mt-n6px" class or whatever.

(I may start using the second option in my code.)

Reply:
Thanks, created  "mt-n6px" class for this